lib/net.js

```diff
@@ -1310,8 +1307,10 @@ Server.prototype._listen2 = function(address, port, addressType, backlog, fd) {
     this.unref();
 
   process.nextTick(emitListeningNT, this);
-};
+}
 
+Server.prototype._setUpListenHandle = setUpListenHandle;
+Server.prototype._listen2 = setUpListenHandle;  // legacy alias
```

Comment:
Removing it would likely (unfortunately) require a semver-major and a deprecation cycle.